devApps/AdalAndroidTestApp/MainActivity.cs

```diff
@@ -121,7 +126,9 @@ protected override void OnActivityResult(int requestCode, Result resultCode, Int
             this.accessTokenTextView.Text = value;
         }
 
+#pragma warning disable AvoidAsyncVoid // Avoid async void
```

Comment:
>#pragma warning disable AvoidAsyncVoid [](start = 0, length = 38)

Seems like for mainform you used the globalsuppression. any reason you are inlining here?